tensorflow/buildspec.yml

```diff
@@ -2,7 +2,7 @@
   region: &REGION <set-$REGION-in-environment>
   framework: &FRAMEWORK tensorflow
   version: &VERSION 2.2.0
-  cuda_version: &CUDA_VERSION cu101
+  cuda_version: &CUDA_VERSION cu102
```

Comment:
@akhilmehra We will have to continue support for CUDA 10.1 images as well, and not just CUDA 10.2.